src/kf_scripts/make_periods_csv.py

```diff
@@ -0,0 +1,52 @@
+import datetime
+import pandas as pd
+import sys
+import os
+
+
+def make_periods_csv(first_day, last_day, stepsize_days, save_dir):
```

Comment:
currently it looks like if you input a time period that is not divisible by `stepsize_days` it will cut off the partial period and not include it in the csv? It might be better to print an error asking the user to either increase or decrease the time period? Or just print a warning that it is not including the remainder?

Reply:
Good point. I think the warning would be useful and most appropriate. I can imagine a user wanting to do weekly inversions for a few months and not caring about some dangling days at the end of the period, but they should definitely be warned.